OktaLogger.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "OktaLogger"
-  s.version          = "1.3.6"
+  s.version          = "1.3.7"
```

Comment:
Why incremented cocoapods release? We have existing 1.3.6 release, why not introduce SPM with 1.3.6?